src/pudl/metadata/fields.py

```diff
@@ -747,6 +772,10 @@
         "description": "Reported units of measure for fuel.",
         # Note: Different ENUM constraints are applied below on EIA vs. FERC1
     },
+    "functional_classification": {
+        "type": "string",
+        "description": "Specifies plant category that record applies to.",
+    },
```

Comment:
Do you know a table where it's used? If not I can do some digging and find one

Reply:
I'm not sure, but I think it's in one of the other tables I did some poking at already, so it would be in the list of "reshaping tables" in #2040.

I think this is probably the first transform to use it. This was more of a forward-looking comment -- that when we come across it next we'll probably want to refer to the same column name and the same string categorizations.